azure-pipelines.yml

```diff
@@ -150,7 +150,7 @@ stages:
     steps:
     - script: ./build.sh --ci
         --configuration $(BuildConfiguration)
-        --prepareMachine
+        --prepareMachine /maxCpuCount:1
```

Comment:
Can we add a tracking issue and a comment here to make sure we remember to remove this in the future? 

Also can we make this only set in macos? That way we wont be setting this in distros that don't need that (and that are more important like Linux) where we could be unintentionally hiding race condition issues.

Reply:
Nevermind, as you pointed out this is only on macos.